### DISCUSSION TITLE
Welcome to CEBRA Discussions!

### DISCUSSION THREAD

**Discussion by MMathisLab** [AdaptiveMotorControlLab/CEBRA#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
* Please feel free to open new discussion questions!

**Comment by MenaVhs** [comment]
Hi, every body! I want to say thank you to create this type of discussions. I just wondering how I can install the program? I just clone the repository. 
Sorry if is a basic question, I'm new with this kind of tools. 
You're great!

**Reply by MMathisLab** [reply]
no problem, right now the source-code is not out, but we can use pip to install 💜, see install docs here: https://cebra.ai/docs/index.html

**Comment by chipsin87** [comment]
Hi Guys,

Read your paper and love your work (nice job!)! Quick question (before I start down a rabbit hole), I have structured time series data (in this case hydrology and rainfall data). I often use methods like UMAP on exponential moving averages to cluster regions of similar hydrological characteristics, would this potentially be a better approach (I got excited when AI read about time series specific non-linear dimension reduction)

**Reply by MMathisLab** [reply]
Hey @chipsin87 - that would likely work very well! You can set time_offset to the same value as your sliding window you used in UMAP to compare directly - measuring consistency of runs would also show then the power of CEBRA -- def. happy to hear how it goes for you, thanks!

**Comment by cgyguy** [comment]
Fanboy here! You're revolutionizing and in my opinion much more exciting than the potential offered by AI. Looking to contribute anyway I can.

**Reply by MMathisLab** [reply]
Thanks!

**Reply by stes** [reply]
Thanks @cgyguy !

**Comment by vvBaronSun** [comment]
Hello, everybody! I read your paper and that’s really an elegant AI model. I could hardly wait to apply CEBRA model to my own research field, but I meet a problem. In fact, there exist two classes of subjects in my data. And every class can be regarded as a kind of “behavior”. Can I use multi-session CEBRA-behavior model to seek the latent embeddings and compare the differences between the two classes of subjects? Looking forward to your reply. Thanks!

**Reply by MMathisLab** [reply]
hi @vvBaronSun no problem :) You can train one model (lets call it control), and another model (lets call it disease), and then compare between the models; check this section and related functions to run to set this up: https://cebra.ai/docs/usage.html#comparing-models 

Then you can use these two classes of models and test new subjects and see how close they are:
https://cebra.ai/docs/usage.html#embeddings-comparison-via-the-infonce-loss
